frontend/tests/components/Header.test.tsx

```diff
@@ -3,18 +3,52 @@ import userEvent from "@testing-library/user-event";
 
 import Header from "src/components/Header";
 
+const props = {
+  logoPath: "/img/logo.svg",
+  primaryLinks: [
+    {
+      i18nKey: "nav_link_home",
+      href: "/",
+    },
+    {
+      i18nKey: "nav_link_health",
+      href: "/health",
+    },
+  ],
+};
+
 describe("Header", () => {
```

Comment:
yay for test coverage:
![image](https://github.com/HHS/grants-equity/assets/25465464/dd0f17d4-25b2-42d6-91f9-d03aa56dd2fe)